content/source/docs/cloud/overview.html.md

```diff
@@ -14,7 +14,7 @@ page_title: "Overview of Features - Terraform Cloud and Terraform Enterprise"
 [terraform enterprise]: /docs/enterprise/index.html
 
 
-> For a hands-on tutorial, try the [Get Started — Terraform Cloud](https://learn.hashicorp.com/terraform/cloud-getting-started/signup?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) guide on HashiCorp Learn.
+> **Hands-on:** Try the [Get Started — Terraform Cloud](https://learn.hashicorp.com/tutorials/terraform/cloud-sign-up?in=terraform/cloud-get-started&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
```

Comment:
Is there a reason why we're linking to the first tutorial in the "Get Started - Terraform Cloud" collection here, but just a link to the "Get Started - Terraform Cloud" collection page in `content/source/docs/cloud/index.html.md`?

Reply:
No, I just slipped on this one 👍🏼 

```suggestion
> **Hands-on:** Try the [Get Started — Terraform Cloud](https://learn.hashicorp.com/collections/terraform/cloud-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn.
```